packages/swapper/src/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts

```diff
@@ -38,8 +39,18 @@ export async function getCowSwapTradeQuote(
     sellAmountIncludingProtocolFeesCryptoBaseUnit,
     potentialAffiliateBps,
     affiliateBps,
+    hasWallet,
   } = input
 
+  // This should never happen, and we have additional checks at execution time re: receiveAddress === zeroAddress but...
+  if (hasWallet && !(receiveAddress && accountNumber !== undefined))
```

Comment:
do we want to add this safety check to arbitrum and cow as well?

Reply:
- Added to Arbitrum in f8ef5825bd0 
- Additional safety for Li.Fi in cc75a5d2bd
- Additional safety for Arb in 7a0e3f7ab9